### DISCUSSION TITLE
请问支持pac规则吗？

### DISCUSSION THREAD

**Discussion by jxyk2007** [apernet/hysteria#845] [Q&A]
通过原有的pac文件和规则格式来让hy2按规则代理

**Comment by tobyxdd** [comment]
PAC 肯定是不会支持的，PAC 是设计给浏览器用的 js 脚本，要支持还得内置 js 引擎。但 Hysteria 有可以实现类似功能的 ACL

**Reply by jxyk2007** [reply]
acl是在服务端设置？还是客户端设置？
